.github/workflows/deploy.yml

```diff
@@ -183,3 +183,35 @@ jobs:
         run : |
           pip install -U twine
           twine upload wheelhouse/*
+  cuQuantum-build:
+    name: Build qiskit-aer-cuQuantum wheels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v1.0.2
+        if: runner.os == 'Windows'
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==2.2.2
+      - name: Build wheels
+        env:
+          CIBW_BEFORE_ALL: "yum install -y yum-utils wget && wget -q https://developer.download.nvidia.com/compute/cuda/11.7.0/local_installers/cuda-repo-rhel7-11-7-local-11.7.0_515.43.04-1.x86_64.rpm && rpm -i cuda-repo-rhel7-11-7-local-11.7.0_515.43.04-1.x86_64.rpm && yum clean all && yum -y install nvidia-driver-latest-dkms cuda && yum -y install cuda-drivers && yum -y install openblas-devel && yum-config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel7/x86_64/cuda-rhel7.repo && yum clean all && yum -y install cuquantum cuquantum-devel cuquantum-doc"
```

Comment:
The nvidia drivers are required to build here? There aren't any GPUs present on the CI node so I typically don't bother installing the drivers.

Reply:
I removed CUDA driver installation and I confirm we do not need CUDA drivers for building binary.